Update generator

michalciolek opened the pull request:
- [x] add `debug` and `release` src dirs
- [x] make `proguard-rules.pro` and `build.gradle` as template
- [x] update npm

ravidsrk commented:
@michalciolek Great work! But you should make the changes on parent repo: https://github.com/androidstarters/kotlin-android-starter

Once you make the update the parent repo, I will just sync it using the sync script https://github.com/androidstarters/generator-kotlin-android/blob/master/generators/app/sync_template_kotlin.js

michalciolek commented:
@ravidsrk I did not find out this script ;D You should add some `contributions` note :)

- Please merge PR https://github.com/androidstarters/kotlin-android-starter/pull/35 where i moved changes from this PR
- Please merge this PR, I fixed some thinks
